src/libpgagroal/configuration.c

```diff
@@ -1291,19 +1297,14 @@ pgagroal_read_limit_configuration(void* shm, char* filename)
                      min_size = max_size;
                   }
 
-                  server_max -= max_size;
-
-                  if (server_max < 0)
-                  {
-                     server_max = 0;
-                     max_size = 0;
-                  }
+		  server_max -= max_size;
```

Comment:
White space

Reply:
somehow my editor is saving spaces as tabs, that causes some problems...